ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -116,7 +116,8 @@ const DropdownBulkSelect = ({
     onSelectNone,
     onChange,
     selectedCnt,
-    totalCnt
+    totalCnt,
+    isDisabled
```

Comment:
Typically these kinds of things are formulated/named positively, as in `isEnabled`, but that's also not important since it's just an internal detail...

Reply:
Here I followed what we set in the lower lever PF component.